app/client/ui/PageWidgetPicker.ts

```diff
@@ -431,6 +459,9 @@ export class PageWidgetSelect extends Disposable {
 
   private _selectType(type: IWidgetType) {
     this._value.type.set(type);
+    if (!isSummaryCompatible(type)) {
+      this._closeSummarizePanel();
+    }
```

Comment:
Is this needed? The form type can't be selected if summarize is enabled, and I can't think of any other scenario where it would need to be closed.